integration-tests/sanity/sanity_test.go

```diff
@@ -188,7 +188,7 @@ var _ = Describe("Sanity", func() {
 
 		It("should delete cstor replica CR", func() {
 			// Getting the cstor replica artifacts
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.CRArtifact)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.CRArtifact)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added